bgpd/bgp_attr.c

```diff
@@ -78,7 +78,7 @@ static const struct message attr_str[] = {
 	{BGP_ATTR_AS_PATHLIMIT, "AS_PATHLIMIT"},
 	{BGP_ATTR_PMSI_TUNNEL, "PMSI_TUNNEL_ATTRIBUTE"},
 	{BGP_ATTR_ENCAP, "ENCAP"},
-#if ENABLE_BGP_VNC_ATTR
+#if ENABLE_BGP_VNC
```

Comment:
These seem completely unrelated to the problem at hand described in the subject line.  What's the goal of this change?

Reply:
you're right - some test code snuck in!

Reply:
should be fixed now